### ISSUE TITLE
Persona 5 Royal Freezes on Android

### ISSUE THREAD

**Issue by Whisskey211** [yuzu-emu/yuzu#12219]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Android - Build 145

### Description of Issue

The game is freezes when play more than 20 - 30 min

![Screenshot_2023-11-29-20-19-13-856_org yuzu yuzu_emu](https://github.com/yuzu-emu/yuzu/assets/106671846/0dae18e0-1bcb-46c2-b1fe-f2b37bb27b96)

### Expected Behavior

No freezes

### Reproduction Steps

Play more than 20 min

### Log File

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/13500987/yuzu_log.txt.old.txt)

### System Configuration

CPU: Snapdragon 7+ Gen2
GPU/Driver: Mesa Turnip R11
RAM: 8gb
OS: Android 13 Miui

**Comment by kongmao45678** [comment]
Try increasing the virtual memory to 8G

**Comment by Whisskey211** [comment]
> Try increasing the virtual memory to 8G

Yes i try to increase virtual memory in ram extension to 5gb with 8gb now i have 13gb ram haha but the game still freeze

**Comment by Whisskey211** [comment]
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13507093/yuzu_log.txt)
This is the new log i recorded
